app/models/automation_request.rb

```diff
@@ -79,4 +66,20 @@ def my_role(_action = nil)
   def log_request_success(_requester_id, _mode)
     # currently we do not log successful automation requests
   end
+
+  def self.parse_out_objects(parameters)
```

Comment:
Should this logic be merged into `build_attrs`?  I don't see any other callers.

Also, I generally don't like the `return thing1, thing2` pattern